core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -521,7 +521,7 @@ public void onClosed(Channel c, IOException cause) {
             channel.addListener(listener);
 
         String slaveVersion = channel.call(new SlaveVersion());
-        log.println("Slave.jar version: " + slaveVersion);
+        log.println("agent.jar version: " + slaveVersion);
```

Comment:
Maybe "Remoting version: " ?
We cannot guarantee the name of the file in this place (actually there may be no file at all)